docs/GettingStarted.md

```diff
@@ -125,7 +125,11 @@ Jest can be used in projects that use [webpack](https://webpack.js.org/) to mana
 
 ### Using Vite
 
-Jest can be used in projects that use [vite](https://vitejs.dev/) to serve source code over native ESM to provide some frontend tooling, vite is an opinionated tool and does offer some out-of-the box workflows. Jest is not fully supported by vite due to how the [plugin system](https://github.com/vitejs/vite/issues/1955#issuecomment-776009094) from vite works, but there are some working examples for first-class jest integration using `vite-jest`, since this is not fully supported, you might as well read the [limitation of the `vite-jest`](https://github.com/sodatea/vite-jest/tree/main/packages/vite-jest#limitations-and-differences-with-commonjs-tests). Refer to the [vite guide](https://vitejs.dev/guide/) to get started.
+Jest is not supported by Vite due to incompatibilities with the Vite [plugin system](https://github.com/vitejs/vite/issues/1955#issuecomment-776009094).
+
+There are examples for Jest integration with Vite in the [vite-jest](https://github.com/sodatea/vite-jest) library. However, as of 5/10/24, this library is not compatible with versions of Vite later than 2.4.2.
+
+One current (5/10/24) alternative is [Vitest](https://vitest.dev/) which has an API intended to be similar to Jest and a [migration guide](https://vitest.dev/guide/migration.html#migrating-from-jest).
```

Comment:
```suggestion
There are examples for Jest integration with Vite in the [vite-jest](https://github.com/sodatea/vite-jest) library. However, this library is not compatible with versions of Vite later than 2.4.2.

One alternative is [Vitest](https://vitest.dev/) which has an API compatible Jest.
```

I don't think it makes sense to link to migration guides in a "getting started" - by definition you have nothing to migrate.